schema/ATHexapod.yaml

```diff
@@ -0,0 +1,68 @@
+$schema: http://json-schema.org/draft-07/schema#
+$id: https://github.com/lsst-ts/ts_salobjATHexapod/blob/master/schema/ATHexapod.yaml
+title: ATHexapod v1
+description: Schema for ATHexapod configuration files
+type: object
+properties:
+  limit_xy_max:
+    description: The max limits of the xy axii
+    type: number
+    default: 22.5
+  limit_z_min:
+    description: The min limits of the z axis
+    type: number
+    default: -12.5
+  limit_z_max:
+    description: The max limits of the z axis
+    type: number
+    default: 12.5
+  limit_uv_max:
+    description: The max limits of the UV axii.
+    type: number
+    default: 7.5
+  limit_w_min:
+    description: The min limit of the W axis.
+    type: number
+    default: -12.5
+  limit_w_max:
+    description: The max limit of the W axis.
+    type: number
+    default: 12.5
+  speed:
+    description: The speed.
+    type: integer
+    default: 1
+  simulator:
```

Comment:
does it really matter for the CSC that it is connecting to the simulator or the real hexapod?